cardano-ping/CHANGELOG.md

```diff
@@ -1,5 +0,0 @@
-# Revision history for cardano-ping
-
-## 0.1.0.0 -- 2022-12-14
-
-* This code was originally from the cardano-ping executable component of the `network-mux` package.
```

Comment:
Instead of removing it we should add `extra-source-files: CHANGELOG.md`.  The same in all other removed `CHANGELOG.md` files.

Reply:
I thought to add, but it seems a lot of these files are never updated.

Reply:
I'm going to go for `CHANGELOG.md` since that's consistent with `interface-CHANGELOG.md`

Reply:
I want to maintain them, I already renamed them in #4249.